gitenberg/config.py

```diff
@@ -78,6 +78,9 @@ def parse(self):
         global data
         data = yaml.load(self.read())
```

Comment:
this means you need a config file even if you're getting the config variables from environment variables, doesn't it?

Reply:
I'm pretty sure the test is testing that case, and it works. I believe it works because the next line initializes it to {} in that case, and then it can still pull stuff out of the environment.

Reply:
doublecheck that you don't have a config file left from a previous use

Reply:
Done